Reqnroll/Events/ExecutionEvent.cs

```diff
@@ -193,20 +214,39 @@ public interface IExecutionOutputEvent
     public class OutputAddedEvent : ExecutionEvent, IExecutionOutputEvent
     {
         public string Text { get; }
+        public FeatureInfo FeatureInfo { get; }
+        public ScenarioInfo ScenarioInfo { get; }
+        public string StepText { get; }
```

Comment:
Maybe I am wrong, but it looks to me that the `StepText` property is not initialize (not used even?)